### ISSUE TITLE
Error running on NixOS-unstable

### ISSUE THREAD

**Issue by Baughn** [rvl/bower2nix#4]
As such:

```
$ git clone https://github.com/rvl/bower2nix
$ nix-build bower2nix
  ( snip )
$ result/bin/bower2nix
module.js:327
    throw err;
    ^

Error: Cannot find module '/nix/store/w3fary8sywpl8ad6p1dkwf5a5yw3rbaq-nodejs-bower2nix-3.0.1/lib/node_modules/bower2nix/bin/../dist/bower2nix'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/nix/store/w3fary8sywpl8ad6p1dkwf5a5yw3rbaq-nodejs-bower2nix-3.0.1/lib/node_modules/bower2nix/bin/bower2nix:3:1)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
```

**Comment by rvl** [comment]
Thanks for reporting. It's not correctly building the javascript in `dist/`. To at least get a build going for dev, try this:

```
cd bower2nix
nix-shell
tsc
nix-build
```

**Comment by rvl** [comment]
Fixed by b47e5b784f1999b945ef7d63e8478a9a52d34304
